website/source/docs/providers/aws/r/instance.html.markdown

```diff
@@ -36,6 +36,8 @@ The following arguments are supported:
      EBS-optimized.
 * `disable_api_termination` - (Optional) If true, enables [EC2 Instance
      Termination Protection](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html#Using_ChangingDisableAPITermination)
+* `instance_initiated_shutdown_behavior` - (Optional) [Shutdown Behavior](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html#Using_ChangingInstanceInitiatedShutdownBehavior) Can be `"stop"`
+or `"terminate"`. (Default: `"stop"`)
```

Comment:
I think this can be updated to something like:

> Defaults to `stop` for EBS-backed instances and `terminate` for instance-store instances. Cannot be set on instance-store instances.